sources/packages/api/src/route-controllers/education-program-offering/education-program-offering.controller.ts

```diff
@@ -283,7 +296,11 @@ export class EducationProgramOfferingController extends BaseController {
         locationId,
         programId,
         programYearId,
-        offeringIntensity,
+        {
+          offeringIntensity: offeringIntensity,
+          offeringStatus: OfferingStatus.Approved,
+          offeringTypes: [OfferingTypes.Public, OfferingTypes.Private],
+        },
```

Comment:
can we create a const for this 
{
          offeringIntensity: offeringIntensity,
          offeringStatus: OfferingStatus.Approved,
          offeringTypes: [OfferingTypes.Public, OfferingTypes.Private],
        },
         with the structure OfferingsFilter type and pass it to the method, so that the method calling will be cleaner.